hardware/raspberrypi/bootmodes/nvme.md

```diff
@@ -135,22 +134,22 @@ enable_uart=1
 uart_2ndstage=1
 ```
 
-Check you have the right version of the start4.elf VideoCore firmware by checking the version of the raspberrypi-bootloader package, it should show a date on or after 3 Mar 2021 in the version string. Run `apt show raspberrypi-bootloader`
+Check you have the right version of the `start4.elf` VideoCore firmware by checking the version of the raspberrypi-bootloader package: it must show a date on or after 3 Mar 2021 in the version string. Run `apt show raspberrypi-bootloader`:
```

Comment:
"a date on or after 3 Mar 2021" -> "a date of 3 Mar 2021 or later" ?

Reply:
Done.